pkg/storageops/utils.go

```diff
@@ -24,3 +28,12 @@ func AddElementToMap(
 		sets[key] = append(sets[key], elem)
 	}
 }
+
+// GetEnvValueStrict fetches value for env variable "key". Returns error if not found or empty
+func GetEnvValueStrict(key string) (string, error) {
+	if val := os.Getenv(key); len(val) != 0 {
+		return val, nil
+	}
+
+	return "", fmt.Errorf("env variable %s is not set", key)
```

Comment:
Q: Is environment variables the only method of getting this information? Would a kv store or file not have them also?

Reply:
Yeah, if we want, we could feed the providers with their configurations through various means (env, file, kv store). To start with, I had to pick one and env variables seemed most natural and easy to use. Also providers like GCE and AWS are written to accept known env variables (just like KUBECONFIG for k8s). Hence went ahead with env variables as the means for setting up configs.